owslib/ows.py

```diff
@@ -219,14 +219,14 @@ def __init__(self, elem, namespace=DEFAULT_OWS_NAMESPACE):
         self.maxx = None
         self.maxy = None
 
-        val = elem.attrib.get('crs')
+        val = elem.attrib.get('crs') or elem.attrib.get('{%s}crs' % namespace)
```

Comment:
Is it possible to update to `elem.attrib.get('{{}}crs'.format(namespace))` to be more Python 3 friendly?